src/test/resources/update/invocation/new.java

```diff
@@ -1,7 +1,5 @@
 package com.diffmin;
 
-import java.util.List;
```

Comment:
@slarse 
> With the import statements in there, tests failed when using a custom pretty-printer.

I had one question. The test failed because the auto-importer removed the unnecessary import statement at it was not being used anywhere?

Reply:
Yep.